devlog/_plan/260827_kiro_builder_id_profile/010_builder_id_request_scoped_profile.md

```
@@ -0,0 +1,156 @@
+# Kiro Builder ID — request-scoped service profile fallback
+
+Unit: `260827_kiro_builder_id_profile` · work-phase `wp1` · opened 2026-08-27
+
+## Symptom
+
+A Kiro request routed to the Builder ID account fails before any tokens are
+generated:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)